tests/core/models/test_collection.py

```diff
@@ -24,30 +24,50 @@
 from core.model.identifier import Identifier
 from core.model.licensing import Hold, License, LicensePool, Loan
 from core.model.work import Work
-from core.testing import DatabaseTest
 from core.util.datetime_helpers import utc_now
 from core.util.string_helpers import base64
+from tests.fixtures.database import DatabaseTransactionFixture
 
 
-class TestCollection(DatabaseTest):
-    def setup_method(self):
-        super().setup_method()
-        self.collection = self._collection(
-            name="test collection", protocol=ExternalIntegration.OVERDRIVE
-        )
+class ExampleCollectionFixture:
```

Comment:
A lot of these could have been dataclasses, no point changing this now since all the code is written now. So.. many.. lines